news/2018-02-22-osu-mania-7k-world-cup-ro16-recap.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "osu!mania 7K World Cup - ro16 Recap"
+date: 2018-02-21 22:00:00 +0000
+---
+
+osu!mania 7K world cup is 
+
+![](/wiki/shared/news/banners/MWC7K_2018_Banner.jpg)
+
+### Navigation
+
+<a href="#mappool">Mappool</a></br><a href="#matches">Tournament Matches</a></br><a href="#ahead">Looking Ahead</a>
+
+--------
+If you've already forgotten the events that have transpired last week, here's a video recap by our beloved Junihuhn summarising the highlights of the Group Stage.
+
+(video link)
+
+This week in the Round of 16, we had 8 scheduled "Best-of-9" matches taking place over a Sunday. Teams this week weren't at risk of elimination, though their performance would significantly impact how they progress through the later stages of this tournament.
+
+There were also some fancy new changes to the stream layout - for the first time in World Cup history, the tournament client resolution was not the standard 4:3, but instead an ultrawide 21:9. With double the width, you can be sure that the highlights were twice as exciting!   
+
+## <a name="mappool" id="mappool"></a>Mappool
+
+**This stage, we saw a total of 39 picks and 14 bans across 7 complete matches.**
+
+### A word from our Mappool Selectors:
+
+*"It's alright but the reception we got from certain maps (namely [Falcom Sound Team J.D.K. - Phantasmagoria](https://osu.ppy.sh/b/293691&m=3) and a little from [KOTOKO - agony (TV Size)](https://osu.ppy.sh/b/261515&m=3)) made me think that we didn't do so well because the patterns weren't really the best in those maps, so now we're aiming for something that can make tournament players be wary of a tough map, instead of having them ban a map outright because of it."* 
```

Comment:
blockquote? (I actually have no clue if it can be used in the news or what it'll look like if so)